modules/ting_search/ting_search.module

```diff
@@ -895,7 +969,7 @@ function ting_search_records_per_page_controls_content() {
   $output = &drupal_static(__FUNCTION__, array());
 
   // Don't show anything if the search result is empty.
-  if (empty($output) && $search_result && $search_result->getNumTotalCollections() > 0) {
+  if (empty($output) && NULL !== $search_result && $search_result->getNumTotalCollections() > 0) {
```

Comment:
Perhaps `!empty($search_result)`?

Reply:
$search_result is either an instance of the result class or null - empty() would imply it could be other values so I'm thinking the null-comparison is the way to go.